src/panels/lovelace/components/hui-image.ts

```diff
@@ -64,13 +68,32 @@ export class HuiImage extends LitElement {
   public connectedCallback(): void {
     super.connectedCallback();
     if (this.cameraImage && this.cameraView !== "live") {
-      this._startUpdateCameraInterval();
+      this._startIntersectionObserverOrUpdates();
```

Comment:
Since we are stopping the observer now on disconnect I left this in but changed it to `_startIntersectionObserverOrUpdates`